### DISCUSSION TITLE
Airtable Integration

### DISCUSSION THREAD

**Discussion by maaaaaann** [Budibase/budibase#1309] [Help] [answered]
Hi everyone, 

I am currently trying to connect my Airtable to Budibase but I am not sure what to do with the API key and base given in Budibase as I cannot find a spot in Airtable where I could insert this. 

Can anyone help? sorry for this question but I am a beginner

**Comment by joebudi** [comment]
Hey @maaaaaann did you get this sorted?

**Comment by gagandeep609** [comment, marked as answer]
Got it from this discussion
https://github.com/Budibase/budibase/discussions/615#discussioncomment-398996

API Key: *Airtable API Key*
Base: *BaseID*

ref:
https://airtable.com/api


Thank you
